doc/src/modules/stats.rst

```diff
@@ -1,5 +1,72 @@
 Stats
 ===========
+SymPy statistics module
+
+Introduces a random variable type into the SymPy language.
+
+Included in this module are the following types :
+
+Finite Types
+
+ * DiscreteUniform
+ * Die
+ * Bernoulli
+ * Coin
+ * Binomial
+ * Hypergeometric
+ * FiniteRV
+ * Rademacher
+ 
+Discrete Types
+
+ * Geometric
+ * Poisson
+ * Logarithmic
+ * NegativeBinomial
+ * YuleSimon
+ * Zeta
+ 
+Continuous Types
+
+ * Arcsin
```

Comment:
Is it possible to make use of `.. automodule: sympy.stats.crv_types`, instead of rewriting them?

The reason being the docstring of respective modules already have the listed types.
See https://github.com/sympy/sympy/blob/master/sympy/stats/crv_types.py#L1-L44 for instance.

Reply:
There is no docstring available for 

https://github.com/sympy/sympy/blob/master/sympy/stats/drv_types.py

I shall create on and reopen a different PR as I messed up this fork with a merge and hard to force push.